Library/Formula/libffi.rb

```diff
@@ -21,15 +16,35 @@ class Libffi < Formula
     depends_on "libtool" => :build
   end
 
-  keg_only :provided_by_osx, "Some formulae require a newer version of libffi."
+  keg_only :provided_by_osx," Some formulae require this version of libffi."
+
+  stable do
+    patch :DATA
```

Comment:
They knew this bug and that the bug is caused form **win32.S**. But they won't accept my advice.